tools/scenario.py

```diff
@@ -42,87 +39,6 @@ def exitBecauseBlueIsNotSupported():
     PlayerId.BLUE: exitBecauseBlueIsNotSupported,
 }
```

Comment:
Maybe out of scope for this PR, but it would be cool if the Python script had no knowledge of the player colors, and Elm somehow just told Python what it needs to do to join the players.

Reply:
Indeed! Future improvement.